### ISSUE TITLE
TensorBoard is "smoothing" lines with two points

### ISSUE THREAD

**Issue by Corbalt** [tensorflow/tensorboard#1127]
If you generate a summary with two points on TensorBoard and apply smoothing, the "smoothed" line will not equal the true line. This is silly. :)

**Comment by nfelt** [comment]
It could probably be smarter in this case but it's quite hard to get a smoothing algorithm that behaves "reasonably" in every single case; there are always tradeoffs.  See #610 for the last smoothing change, and #786 for another issue with the current algorithm.

Quoting Dandelion from https://github.com/tensorflow/tensorboard/issues/610#issuecomment-334380578: "It seems that every change in the smoothing algorithm results in new tradeoffs and biases, and new requests to change the algorithm. At this point it's not a priority for the team, so just filing an issue probably won't directly result in changes. However, the smoothing algorithm is actually a very simple piece of code to change. If you come up with a new algorithm, and demonstrate persuasively why it is an improvement over the current one (taking into account new biases and edge cases that it introduces), I'll happily merge it."

**Comment by jart** [comment]
@Corbalt I agree and I'm interested in knowing if a use case exists for having two samples in a scalars chart?

**Comment by Corbalt** [comment]
If you train a tf.estimator model with default settings, it performs eval every 10 minutes. If your training takes less than 10 minutes, you end up with eval plots with two points: one from the start of training and one from the end. This was the situation I was in when I pulled up TB and saw the plot. I was surprised at my metrics being low, but realized it was due to the smoothed line being inaccurate.

**Comment by bileschi** [comment]
Setting startup smoothing to zero would also solve this problem.  See https://github.com/tensorflow/tensorboard/issues/294
